styles/globals.css

```diff
@@ -1,3 +1,6 @@
 :root {
-  --border-color: #413f3f
-}
\ No newline at end of file
+  --border-color: #413f3f;
+  --primary-color: #1d1283;
```

Comment:
Is this primary color variable used? Didn't see in new changes.

Reply:
will use it in navbar